### PR TITLE
minor update to ERE Event reader, other small fixes

### LINKED ISSUE #420: update NER documentation: 6G needed

### LINKED ISSUE #441: ner readme gives outdated command
command has classpath "dist/*:lib/*..." which refers to standalone packaging setup.

### LINKED ISSUE #446: [pipeline] script doesn't exist.
In the pipeline docs:
https://github.com/CogComp/cogcomp-nlp/tree/master/pipeline#running-a-simple-command-line-test
 `runPreprocessor.sh` is mentioned, but it doesn't exist in the scripts folder.

### PR DESCRIPTION
**Pull request by mssammon**
reads extra attribute specified in TAC event argument linking data (`origin`). Updates NER README.md. Adds missing pipeline script and test files. Closes #420. Closes #441. Closes #446.

### CODE REVIEW

File: big-data-utils/pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <parent>
         <artifactId>illinois-cogcomp-nlp</artifactId>
         <groupId>edu.illinois.cs.cogcomp</groupId>
-        <version>3.1.16</version>
+        <version>3.1.18</version>
```
Comment (danyaljj) [inline big-data-utils/pom.xml]: I think you have to increment the version again.

**Comment by danyaljj** [comment]
Thanks Mark. I can see that you updated the pipeline script (which is good), but I think #446 is asking something different. The script name mentioned in the readme doesn't exist.

**Comment by mssammon** [comment]
Now addresses #446 ; pipeline readme now refers to correctly named scripts, assuming maven module structure.

File: pipeline/README.md [hunk 2]
```diff
@@ -98,18 +93,15 @@ implement a class that follows the `Tokenizer` interface from
...
-pipeline works after you have downloaded
-it. `scripts/runPreprocessor.sh` takes as arguments a configuration file
+NOTE: These commands assume you ran `mvn install` and `mvn dependency:copy-dependencies`,
+which create the pipeline binary in `target/` and copies all dependency jars into 
+`target/dependency`.  
+Two sample scripts are provided to test that the pipeline works after you have downloaded
+it. `scripts/runPipelineOnDataset.sh` takes as arguments a configuration file
 and a text file; it processes the text file according to the
 properties set in the config file, and writes output to STDOUT.
 scripts/testPreprocessor.sh is a self-contained script that calls
```
Comment (danyaljj) [inline pipeline/README.md]: Minor: add single quotes?  `scripts/testPreprocessor.sh` is a self-contained ....

**Comment by danyaljj** [comment]
Looks good to me. Merge call with @cowchipkid

**Comment by cowchipkid** [comment]
I will look at this on Thurs, Thurs and Friday are my Dan days.

File: pipeline/README.md (hunk 2 above)
Comment (mssammon) [inline pipeline/README.md]: fixed.

**Pull request merged**